CHANGELOG.md

```diff
@@ -120,6 +120,9 @@ Every module contains its own CHANGELOG.md. Please refer to the module you are i
 
 ### API Breaking Changes
 
+* (client) [#20976](https://github.com/cosmos/cosmos-sdk/pull/20976) Simplified command initialization by removing unnecessary parameters such as `txConfig` and `addressCodec`.
+  * Remove parameter `txConfig` from `genutilcli.Commands`,`genutilcli.CommandsWithCustomMigrationMap`,`genutilcli.GenTxCmd`.
+  * Remove parameter `addressCodec` from `genutilcli.GenTxCmd`,`genutilcli.AddGenesisAccountCmd`,`stakingcli.BuildCreateValidatorMsg`.
```

Comment:
Can we check if addressCodec was there in 0.50? I don't think so, and that means we can remove this line.

Reply:
I checked the latest version 0.50, and it does have addressCodec.

https://github.com/cosmos/cosmos-sdk/blob/v0.50.8/x/genutil/client/cli/commands.go